packages/datagrid/src/components/DefaultCellRenderer/DefaultCellRenderer.component.js

```diff
@@ -11,6 +11,9 @@ import theme from './DefaultCell.scss';
 export const CELL_RENDERER_COMPONENT = 'cellRenderer';
 
 export default function DefaultCellRenderer({ avroRenderer, colDef, value, getComponent }) {
+	if (value === undefined) {
+		return null;
```

Comment:
should not we render empty div ?
or even better support a props with noValueMessage ?
What if the value === null ? or is a string ?
seems lots of case are not covered here